src/app/store/selectors/app.selectors.ts

```diff
@@ -103,6 +103,17 @@ export const isAdmin = createSelector(
   state => state.user.isAdmin
 );
 
+export const sidenav = createSelector(
```

Comment:
the name is too generic as for me, maybe we call it "sidenavState" or something else to express what it does